apps/root/src/pages/strategy-guardian-detail/strategy-management/components/delay-withdraw-warning/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { colors, ContainerBox, InfoCircleIcon, TimerIcon, Tooltip, Typography } from 'ui-library';
+
+const StyledDelayWithdrawWarningContainer = styled(ContainerBox)`
+  ${({ theme: { palette } }) => `
+        border-bottom: 1.5px solid ${colors[palette.mode].border.border1};
+    `}
+`;
+
+const ButtonIconContainer = styled(ContainerBox)`
+  ${({ theme: { palette, spacing } }) => `
+    padding: ${spacing(1.125)};
+    border-radius: 50%;
+    background-color: ${colors[palette.mode].background.tertiary};
+  `}
+`;
+
+const DelayWithdrawWarning = () => (
+  <StyledDelayWithdrawWarningContainer gap={1} alignItems="center">
+    <ButtonIconContainer>
+      <TimerIcon
+        sx={({ spacing }) => ({
```

Comment:
we could just send fontSize as the prop, since it already handles that the svg icon

Reply:
fontSize is theme typed, so custom values (like 15px in this case) would not be accepted